doc/source/all-styles.rst

```diff
@@ -0,0 +1,46 @@
+
+Style
+=====
+
+In the PyAnsys ecosystem, three key styles—packaging, coding, and documentation—
```

Comment:
```suggestion
In the PyAnsys ecosystem, three key styles —packaging, coding, and documentation—
```

Reply:
@RobPasMue @dipinknair Google dev style guideline says no spaces before dashes. (This is also true in other style guides.) I generally prefer using commas over dashes: "In the PyAnsys ecosystem, three key styles, packaging, coding, and documentation, ...." Vale will throw an error if the spaces appear before the dashes.

Reply:
Oh, didn't know that!